src/metabase/models/login_history.clj

```diff
@@ -48,7 +48,9 @@
   :type       :boolean
   :visibility :internal
   :setter     :none
-  :default    true)
+  :default    true
+  :doc "This variable controls the geocoding service that Metabase uses to know the location from where your users logged in.
+        Setting this variable to false also disables this reverse geocoding functionality.")
```

Comment:
This seems like it's describing a different setting?

Reply:
I ported that elaboration from the existing docs. Looks like @paoliniluis added it a while back: https://github.com/metabase/metabase/pull/20804.

@paoliniluis is this comment still correct and useful?

Reply:
Oh, I didn't get how this was related to geocoding. The `also` in the original docs seems important